### ISSUE TITLE
Support comma in subtitle string when (not json)

### ISSUE THREAD

**Issue by martinvandiemen** [swiftDialog/swiftDialog#349]
**Is your feature request related to a problem? Please describe.**
The new subtitle is great but it doesn't support comma like progress text does. It will break the line remove the part starting from the comma. We need to be able to append the string to the command file instead of the JSON (because of the workflow we have).

**Describe the solution you'd like**
Support comma like in progress text.

**Describe alternatives you've considered**
I tried a fullwidth comma but this isn't passed through to the client correctly.

**Additional context**
-

**Comment by bradtchapman** [comment]
Can you attach screenshots and sample configuration code to this issue?

**Comment by martinvandiemen** [comment]
For example.

**Subtitle**
> Microsoft Defender is an endpoint security platform designed to help prevent, detect, investigate, and respond to advanced threats.

**Result**
> Microsoft Defender is an endpoint security platform designed to help prevent

<img width="932" alt="Screenshot 2024-02-15 at 16 52 44" src="https://github.com/swiftDialog/swiftDialog/assets/946622/51f6019e-4af3-4814-811c-e4be2680de21">

**Comment by bradtchapman** [comment]
@martinvandiemen : it thinks ", detect" is the next argument in the command.  Try putting it in single quotes or double quotes.

**Comment by martinvandiemen** [comment]
@BigMacAdmin  @bradtchapman this is not a bug, it's a feature request. Like @bartreardon meantioned it's not supported currently: https://github.com/swiftDialog/swiftDialog/issues/331#issuecomment-1905339553. Quotes around it, escaping comma, or whatsover doesn't helpt.

**Comment by bartreardon** [comment]
yeah, it's because commas are being used as the separator to split one argument into multiple related properties. It's not doing anything smart about it, simply splitting on comma and processing the pieces so in a subtitle it'll see a comma and consider that the end of the subtitle property.

**Comment by martinvandiemen** [comment]
@bartreardon would it be possible to ignore the comma if it's between quotes?
